collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_pgsnap.py

```diff
@@ -307,15 +307,6 @@ def restore_pgsnapvolume(module, array):
         + "."
         + module.params["restore"]
     )
-    if "::" in module.params["target"]:
```

Comment:
I need to check on which version this requirement changed and then we need to add in a API version check so that people running older versions of Purity still get this error.

Reply:
Added API version check